source/grid/grid_in.cc

```diff
@@ -3113,6 +3640,13 @@ GridIn<dim, spacedim>::read(std::istream &in, Format format)
                           "functions, instead."));
         return;
 
+      case exodusii:
+        Assert(false,
```

Comment:
That is unfortunate and wouldn't be the case if you make the mapping argument optional or a return value.

Reply:
I don't understand your comment. The ExodusII API requires a filename so this function call can't work with ExodusII for the same reason that it can't work for assimp.

Reply:
Oh, I am sorry. I thought the reason is the second argument. Ignore this comment then.